src/bci_build/package/appcollection.py

```diff
@@ -0,0 +1,46 @@
+"""Application Collection Containers that are generated with the BCI tooling"""
+
+from dataclasses import dataclass
+
+from bci_build.package import ApplicationStackContainer
+from bci_build.package import Arch
+
+
+@dataclass
+class ApplicationCollectionContainer(ApplicationStackContainer):
+    """Containers for the Rancher Application Collection Distribution Platform."""
+
+    @property
+    def _registry_prefix(self) -> str:
+        if self.is_opensuse:
+            return super()._registry_prefix
+        return "containers"
+
+    @property
+    def registry(self) -> str:
+        if self.is_opensuse:
+            return super().registry
+        return "dp.apps.rancher.io"
+
+    @property
+    def url(self) -> str:
+        if self.is_opensuse:
+            return super().url
+        return f"https://apps.rancher.io/applications/{self.name}"
+
+    @property
+    def title(self) -> str:
+        if self.is_opensuse:
+            return super().title
+        return self.pretty_name
+
+    @property
+    def eula(self) -> str:
+        if self.is_opensuse:
+            return "sle-bci"
+        return "sle-eula"
+
```

Comment:
This is exactly the same code as in `ApplicationStackContainer`
```suggestion
```

Reply:
@dcermak  the patch wasn't merged yet when I wrote this code, hence the duplication.